<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Main_Window</class>
 <widget class="QMainWindow" name="Main_Window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Sprite_Editor</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="previousFrame">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>100</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
      <kerning>true</kerning>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="text">
     <string>Previous Frame</string>
    </property>
   </widget>
   <widget class="QPushButton" name="nextFrame">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>10</y>
      <width>100</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Next Frame</string>
    </property>
   </widget>
   <widget class="QPushButton" name="newFrame">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>70</y>
      <width>110</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>New Frame</string>
    </property>
   </widget>
   <widget class="QPushButton" name="previewSprite">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>310</y>
      <width>170</width>
      <height>70</height>
     </rect>
    </property>
    <property name="text">
     <string>Preview Sprite</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>140</y>
      <width>210</width>
      <height>150</height>
     </rect>
    </property>
    <property name="title">
     <string>Tools</string>
    </property>
    <widget class="QToolButton" name="colorPickerButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>91</width>
       <height>111</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>ArrowCursor</cursorShape>
     </property>
     <property name="text">
      <string>Brush Color</string>
     </property>
    </widget>
    <widget class="QToolButton" name="eraseButton">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>20</y>
       <width>91</width>
       <height>111</height>
      </rect>
     </property>
     <property name="text">
      <string>Erase</string>
     </property>
    </widget>
   </widget>
   <widget class="PixelEditor" name="pixelEditor" native="true">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>20</y>
      <width>500</width>
      <height>500</height>
     </rect>
    </property>
   </widget>
   <widget class="QCheckBox" name="actualSizeCheckBox">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>400</y>
      <width>141</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Preview Actual Size?</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="actionSave"/>
    <addaction name="actionLoad"/>
    <addaction name="actionClose"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionNew">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>New</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionLoad">
   <property name="text">
    <string>Load</string>
   </property>
  </action>
  <action name="actionClose">
   <property name="text">
    <string>Close</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PixelEditor</class>
   <extends>QWidget</extends>
   <header>pixel_editor.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
